wandb/board/ui/src/components/RunSummary.js

```diff
@@ -107,8 +107,9 @@ class RunSummary extends Component {
                   </span>
                 )}
               </Grid.Column>
-              <Grid.Column width={6} textAlign="right">
-                run{' '}
+              <Grid.Column width={8} textAlign="right">
+                tags {bucket.tags.map(tag => <Label key={tag}>{tag}</Label>)}
+                &nbsp;&nbsp;&nbsp;&nbsp; run{' '}
```

Comment:
Heh, nice. Could use <span> and inline css here.

(you can do like <span style={{marginLeft: 10}}>content</span>

Reply:
Was just copying the example nearby. What do you mean "like content"? Removing the spaces because they've been removed from the adjacent code that I copied.